Rest.Models/Inventory.cs

```diff
@@ -0,0 +1,6 @@
+public class Inventory
```

Comment:
Isn't better to use record classes for this?

What do you think @Aryevang ?

Reply:
Is way better to use record classes than normal classes @reynoldmorel/@Lcv004. The performance and the security due to immutability are the key feature.

Reply:
Understood! I already did some research about records and made changes.

Reply:
@Lcv004 I still see `Inventory` as a `class` instead of `record`. Did you change anything or you'll keep it like that?

If so, please explain why.

Reply:
Sorry, I forgot to add my changes. Already done!